core/templates/pages/about-page/about-page.controller.ts

```diff
@@ -54,6 +56,9 @@ angular.module('oppia').directive('aboutPage', [
           ctrl.TAB_ID_ABOUT = 'about';
           ctrl.TAB_ID_FOUNDATION = 'foundation';
           ctrl.TAB_ID_CREDITS = 'credits';
+          ctrl.ALPHABETS = [
+            'A', 'B', 'C', 'D', 'E', 'F', 'G', 'H', 'I', 'J', 'K', 'L', 'M',
+            'N', 'O', 'P', 'Q', 'R', 'S', 'T', 'U', 'V', 'W', 'X', 'Y', 'Z'];
```

Comment:
Isn't this just one alphabet? Also, why separate the letters if they can be written as `'ABC…XYZ'`.

Reply:
Sorry, I did not understand your query. Basically, letters are added as a list to enable `ng-repeat` on this list and use it to list down developer names starting with a letter on the credits page.

Reply:
```suggestion
          ctrl.ALPHABET = 'ABCDEFGHIJKLMNOPQRSTUVWXYZ';
```
since this is just one alphabet and we don't need to have the letters separately.

Reply:
Sorry, why don't we need to have them separately? It is required for ng-repeat [here](https://github.com/oppia/oppia/pull/9219/files#diff-8c66287c207757ecd004b456310aa006R106).

Reply:
Oh, okay, sorry this was misunderstanding of AngularJS from my side I thought arrays and strings are ± the same but apparently are not. Still I would use `ALPHABET` as we have one aphabet and nt multiple alphabets here.

Reply:
Updated to `ALPHABET_LIST`.